AppInsightsCommon/package.json

```diff
@@ -1,18 +1,30 @@
 {
-    "name": "applicationinsights-common-amd",
-    "version": "0.0.99-beta",
-    "description": "Microsoft Application Insights Common JavaScript Library - AMD version",
-    "main": "./bundle/applicationinsights-common.js",
-    "types": "./bundle/applicationinsights-common.d.ts",
+    "name": "@microsoft/applicationinsights-common",
+    "version": "1.0.0-beta-2",
+    "description": "Microsoft Application Insights Common JavaScript Library",
```

Comment:
So, from Brian's presentation, there is another folder dist for node. Was this not in the example PR?